topdown/encoding.go

```diff
@@ -258,6 +267,31 @@ func builtinYAMLIsValid(a ast.Value) (ast.Value, error) {
 	return ast.Boolean(err == nil), nil
 }
 
+func builtinHexEncode(a ast.Value) (ast.Value, error) {
+	str, err := builtins.StringOperand(a, 1)
+	if err != nil {
+		return nil, err
+	}
+	src := []byte(str)
+	dst := make([]byte, hex.EncodedLen(len(src)))
+	hex.Encode(dst, src)
```

Comment:
❓ I believe `encoding/hex` is similar to `encoding/base64`. So why not use the [`hex.EncodeToString`](https://golang.org/pkg/encoding/hex/#EncodeToString) method here?

Reply:
Great observation, thank you. I've updated.